pkg/lvgl/Makefile.dep

```diff
@@ -0,0 +1,18 @@
+USEMODULE += lvgl_core
+USEMODULE += lvgl_draw
+USEMODULE += lvgl_font
+USEMODULE += lvgl_hal
+USEMODULE += lvgl_misc
+USEMODULE += lvgl_objx
+USEMODULE += lvgl_themes
+
+USEMODULE += lvgl_contrib
+USEMODULE += disp_dev
+USEMODULE += core_thread_flags
+USEMODULE += xtimer
```

Comment:
These should be a dependency of lvgl_contrib and not of the lvgl package itself I think.

Reply:
Done